src/github.com/couchbase/sync_gateway/base/sharded_sequence_clock.go

```diff
@@ -35,19 +36,40 @@ type PartitionStorage struct {
 	VbNos []uint16 `json:"vbNos"`
 }
 
+type PartitionStorageSet []PartitionStorage
+
+// Sorts the PartitionStorageSet by Uuid
+func (c PartitionStorageSet) Sort() {
+	sort.Sort(c)
+}
+
+// Implementation of sort.Interface
+func (c PartitionStorageSet) Len() int           { return len(c) }
+func (c PartitionStorageSet) Less(i, j int) bool { return c[i].Uuid < c[j].Uuid }
+func (c PartitionStorageSet) Swap(i, j int)      { c[i], c[j] = c[j], c[i] }
+
+func (c PartitionStorageSet) String() string {
+	result := ""
+	c.Sort()
+	for _, partition := range c {
+		result = fmt.Sprintf("%s[%d - %v]", result, partition.Index, partition.VbNos)
+	}
+	return result
+}
```

Comment:
A bit confused .. result will get clobbered except for last iteration.  Why not just get last element of c?